### PR TITLE
chore(artwork lists): remove old saved artworks implementation

### PR DESCRIPTION
**Pull request by anandaroop**
The type of this PR is: **Chore**

This PR solves [ONYX-4]

Follow up to https://github.com/artsy/force/pull/12938

### Description

- Removes unused code from the previous implementation
- Removes code that was apparently still 🙈 [in use and accessible](https://artsy.slack.com/archives/C05EQL4R5N0/p1695764545466069), from an even _earlier_ implementation (back when Saved Artworks lived at `/settings/saves`)
- Because of that^ adds a redirect from `/settings/saves` → `/collector-profile/saves`

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ONYX-4]: https://artsyproduct.atlassian.net/browse/ONYX-4?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by anandaroop** [comment]
Love to see it…

diff

File: src/Apps/Settings/settingsRoutes.tsx [hunk 1]
```diff
@@ -201,18 +191,9 @@ export const settingsRoutes: AppRouteConfig[] = [
       },
       {
         path: "saves",
-        getComponent: () => SavesRoute,
-        onClientSideRender: () => {
-          SavesRoute.preload()
+        render: () => {
+          throw new RedirectException(`/collector-profile/saves`, 301)
```
Comment (nickskalkin) [inline src/Apps/Settings/settingsRoutes.tsx:195]: 👍

**Review by nickskalkin — APPROVED** [review]
awesome!

**Review by dblandin — APPROVED** [review]
🚀

**Pull request merged**